aggregator/options.go

```diff
@@ -612,7 +698,15 @@ func (o *options) computeFullGaugePrefix() {
 
 // By default we use e.g. ".p50", ".p95", ".p99" for the 50th/95th/99th percentile.
 func defaultTimerQuantileSuffixFn(quantile float64) []byte {
-	return []byte(fmt.Sprintf(".p%0.0f", quantile*100))
+	return []byte(".p" + addSuffixZero(strconv.FormatFloat(quantile, 'f', -1, 64)[2:]))
```

Comment:
Add some comments here explaining what is being done? Also might wanna get rid of the leading zeros after the decimal point (e.g., `p5` is probably better than `p05`)